domain/src/main/java/com/softserve/borda/dto/TagDTO.java

```diff
@@ -8,4 +8,5 @@
     private Long id;
     private String text;
     private String color;
+    private Long boardId;
```

Comment:
As I understand, we can add a tag to a ticket also, why ticket id is missed here?

Reply:
Do we have a relation between ticket and tag? - https://github.com/nelfan/BORDA/blob/39db6bea7c22da5dbd236480414b78eb44df15bf/service/src/main/java/com/softserve/borda/services/impl/TicketServiceImpl.java#L86